docs/docs/firefly-iii/api.md

```diff
@@ -41,3 +41,11 @@ Copy the entire token. Yes, it's very long!
 
 ![Copy the entire token.](images/pat-long.png)
 
+To use the token you have to pass an `Authorization: Bearer <token>` HTTP header. As an example in curl:
+
+```bash
+curl -X GET 'https://[firefly domain name/api/v1/webhooks' \
```

Comment:
It's "Firefly III", not "firefly". How about this?

`curl -X GET 'https://your-domain/api/v1/webhooks' \`